modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js

```diff
@@ -299,7 +299,7 @@ function addActions(container) {
         $("#message").append(confirmationDialog);
         message({
             title: "Consent Confirmation",
-            content: 'Are you sure you want to revoke this consent? this is not reversible...',
+            content: 'Are you sure you want to revoke this consent? This is not reversible.',
```

Comment:
Isn't the correct word Irreversible ?. Not reversible sounds more user friendly ?